.github/workflows/deploy-lambdas.yaml

```diff
@@ -78,20 +78,18 @@ jobs:
       - uses: actions/checkout@v4
       - name: Build Docker image
         working-directory: ./lambdas/${{ matrix.path }}
-        run: |
-          image_name=quiltdata/lambdas/${{ matrix.path }}:${{ github.sha }}
-          docker buildx build --platform=linux/amd64 -t "$image_name" -f Dockerfile ..
+        run: docker build -t "quiltdata/lambdas/${{ matrix.path }}:${{ github.sha }}" -f Dockerfile ..
```

Comment:
are you sure `docker build` and `docker buildx` do exactly the same thing?

Reply:
I mean I thought `buildx` is super-duper and works faster.

Reply:
`buildx` just has extra features like cross-compiling for a different OS/arch (so useful for building Linux images on a Mac, or maybe specifically Arm64 Macs?) But looks like it also does parallel builds, so I guess that's a reason to use it here.